crawl-ref/source/dat/des/variable/large_themed.des

```diff
@@ -2316,3 +2316,85 @@ xxxx-xx..xx....xx-xxx-x
 xxxxxxxxx-x.....xxxxxxx
 xxxxxxxxxxx.@@@..xxxxxx
 ENDMAP
+
+NAME:   dreamdust_cave_troll
+TAGS:   no_monster_gen
+ORIENT: northwest
+DEPTH:  D:10-, Lair
+NSUBST: - = 1 / 2=? / .
+KFEAT:  ^ = web
+MONS:   deep troll
+: dgn.delayed_decay(_G, "?", "yak corpse")
+MARKER: ! = lua:fog_machine { cloud_type='flame', pow_min = 10, pow_max = 10, \
+                              delay = 10, size = 1, excl_rad = 0 }
+MAP
+xxxxxxxxxx
+x--xxxx
+x-!-xxxx
+xx-.xxxxx
+xxxx.^xxx
+xxxxx.xx
+xxx^.xx
+xxx@xx
+x
+ENDMAP
+
+NAME:   dreamdust_under_siege
+TAGS:   transparent no_monster_gen no_pool_fixup
```

Comment:
This is probably a vault that wants `patrolling`, since it's trying to establish a theme of defenders under siege. If the "attacking" monsters wander off after level generation, the player will just encounter a fortress, potentially even an empty one if defenders also wander off.